test/quantization/eager/test_numeric_suite_eager.py

```diff
@@ -421,14 +421,14 @@ def test_compare_model_outputs_functional_static(self):
         q_model(self.img_data_2d[0][0])
         q_model = convert(q_model)
         act_compare_dict = compare_model_outputs(model, q_model, self.img_data_2d[0][0])
-        self.assertEqual(len(act_compare_dict), 7)
+        self.assertEqual(len(act_compare_dict), 5)
         expected_act_compare_dict_keys = {
             "mycat.stats",
             "myadd.stats",
             "mymul.stats",
             "myadd_relu.stats",
-            "my_scalar_add.stats",
-            "my_scalar_mul.stats",
+            # "my_scalar_add.stats", # empty
```

Comment:
is this change relevant?

Reply:
+1, what's the context for this?

Reply:
this tensor is empty, it would be filtered after this PR.

Reply:
in that case, I think it would be better to delete the lines instead of commenting them out

Reply:
sure